tests/test_filter_count.py

```diff
@@ -332,40 +337,87 @@ def test_pruning_order_method():
 @pytest.mark.parametrize(
     "filter_by", ["prune_by_noise_rate", "prune_by_class", "both", "confident_learning"]
 )
-def test_find_label_issues_multi_label(multi_label, filter_by):
+@pytest.mark.parametrize(
```

Comment:
We may want extra checks to ensure informative error message gets printed if the user wrongly passes in regular labels with `multi_label=True` (or vice versa).   Currently I get:
```
>>> filter.find_label_issues([0,0,0,1,1,1],pred_probs=np.array([[0.1,0.9],[0.1,0.9],[0.1,0.9],[0.1,0.9],[0.1,0.9],[0.1,0.9]]), return_indices_ranked_by='self_confidence', multi_label=True)

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/jonas/Dropbox/Cleanlab/opensource/pr_review/cleanlab/cleanlab/filter.py", line 242, in find_label_issues
    return _find_label_issues_multilabel(
  File "/Users/jonas/Dropbox/Cleanlab/opensource/pr_review/cleanlab/cleanlab/filter.py", line 528, in _find_label_issues_multilabel
    y_one = int2onehot(labels)
  File "/Users/jonas/Dropbox/Cleanlab/opensource/pr_review/cleanlab/cleanlab/internal/util.py", line 243, in int2onehot
    return mlb.fit_transform(labels)
  File "/Users/jonas/virtual/cldev/lib/python3.9/site-packages/sklearn/preprocessing/_label.py", line 794, in fit_transform
    yt = self._transform(y, class_mapping)
  File "/Users/jonas/virtual/cldev/lib/python3.9/site-packages/sklearn/preprocessing/_label.py", line 867, in _transform
    for label in labels:
TypeError: 'int' object is not iterable
```

Reply:
Can you point to where you are resolving comments like these before you mark them resolved? Otherwise it is not so easy for reviewers to figure it out

Reply:
Resolved here: https://github.com/aditya1503/cleanlab/blob/682bd91957f54191add1c892e39714fb6a05af6e/cleanlab/filter.py#L532